modules/user_accounts/php/my_preferences.class.inc

```diff
@@ -191,8 +191,8 @@ class My_Preferences extends \NDB_Form
         // update the user
         $success = $user->update($set);
 
-        // Now set the password. Note that this field is named incorrectly at this point in the code
-        // and represents a plaintext password, not a hash.
+        // Now set the password. Note that this field is named incorrectly
```

Comment:
Made this change because travis was failing from line length exceeding 85 characters.